### DISCUSSION TITLE
账号校验错误

### DISCUSSION THREAD

**Discussion by bogedan** [Mufanc/iSmartAuto#38] [Q&A]
在没有对PC上的参数与数据进行修改的情况下，使用同样的账号进行刷课时发现，在flash - i后复制课程ID回车时总出现账号校验报错的报错行，一段时间后又恢复正常，求解

**Comment by AllBlueIce** [comment]
请问是哪里出了问题 我也出现这个问题了

**Reply by bogedan** [reply]
开代理了
